Formula/yarn.rb

```diff
@@ -18,9 +16,10 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
-    inreplace "#{libexec}/lib/node_modules/yarn/package.json", '"installationMethod": "tar"', '"installationMethod": "homebrew"'
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarn"
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarnpkg"
```

Comment:
In general @Daniel15 having very strong feelings on this PR either way would be valued input.